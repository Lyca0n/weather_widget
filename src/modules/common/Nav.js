import React from 'react';
import { NavLink } from 'react-router-dom'

const Nav =()=>(
    <div className="nav">
        <ul className="nav__list">
        <li className="nav__link">
            <NavLink to="/">Home</NavLink>
        </li>
    
        </ul>
    </div>
);

export default Nav;