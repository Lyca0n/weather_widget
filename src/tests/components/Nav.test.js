import React from 'react';
import { shallow } from 'enzyme';
import Nav from '../../modules/common/Nav';

test('should render Nav correctly', () => {
  const wrapper = shallow(<Nav />);
  expect(wrapper).toMatchSnapshot();
});

