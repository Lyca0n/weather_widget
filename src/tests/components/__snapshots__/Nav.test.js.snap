// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Nav correctly 1`] = `
<div
  className="nav"
>
  <ul
    className="nav__list"
  >
    <li
      className="nav__link"
    >
      <NavLink
        to="/"
      >
        Home
      </NavLink>
    </li>
  </ul>
</div>
`;
